### ISSUE TITLE
See why some PURLs have missing fulltext source

### ISSUE THREAD

**Issue by marlo-longley** [sul-dlss/vt-arclight#236]
```sh
      01 W, [2022-11-17T11:34:18.472202 #2375653]  WARN -- : Missing fulltext source: /opt/app/vt/vt/releases/20221117193329/data/fulltext/qx410zd8457.txt
      01 W, [2022-11-17T11:34:18.486598 #2375653]  WARN -- : Missing fulltext source: /opt/app/vt/vt/releases/20221117193329/data/fulltext/mm067zg9616.txt
      01 W, [2022-11-17T11:34:18.501262 #2375653]  WARN -- : Missing fulltext source: /opt/app/vt/vt/releases/20221117193329/data/fulltext/yg944nv6818.txt
      01 W, [2022-11-17T11:34:18.516331 #2375653]  WARN -- : Missing fulltext source: /opt/app/vt/vt/releases/20221117193329/data/fulltext/bz584xw5468.txt
      01 W, [2022-11-17T11:34:18.536064 #2375653]  WARN -- : Missing fulltext source: /opt/app/vt/vt/releases/20221117193329/data/fulltext/wx320nk2472.txt
      01 W, [2022-11-17T11:34:18.550066 #2375653]  WARN -- : Missing fulltext source: /opt/app/vt/vt/releases/20221117193329/data/fulltext/vg502jk7717.txt
      01 W, [2022-11-17T11:34:18.565261 #2375653]  WARN -- : Missing fulltext source: /opt/app/vt/vt/releases/20221117193329/data/fulltext/kz286zs1464.txt
      01 W, [2022-11-17T11:34:18.579024 #2375653]  WARN -- : Missing fulltext source: /opt/app/vt/vt/releases/20221117193329/data/fulltext/yc515xf7934.txt
      01 W, [2022-11-17T11:34:18.592642 #2375653]  WARN -- : Missing fulltext source: /opt/app/vt/vt/releases/20221117193329/data/fulltext/zr106xw6265.txt
      01 W, [2022-11-17T11:34:18.613158 #2375653]  WARN -- : Missing fulltext source: /opt/app/vt/vt/releases/20221117193329/data/fulltext/mb937hy9472.txt
      01 W, [2022-11-17T11:34:18.626719 #2375653]  WARN -- : Missing fulltext source: /opt/app/vt/vt/releases/20221117193329/data/fulltext/ky983nx7353.txt
      01 W, [2022-11-17T11:34:18.641741 #2375653]  WARN -- : Missing fulltext source: /opt/app/vt/vt/releases/20221117193329/data/fulltext/fp989jq4871.txt
      01 W, [2022-11-17T11:34:18.641966 #2375653]  WARN -- : Missing fulltext source: /opt/app/vt/vt/releases/20221117193329/data/fulltext/qx410zd8457.txt
```

**Comment by marlo-longley** [comment]
We need to reharvest the fulltext from the latest EAD.

**Comment by thatbudakguy** [comment]
resolved by #269?

**Comment by marlo-longley** [comment]
I'd say we resolved the reason behind this yeah, and the work of harvest/index is ongoing in #269. Good call @thatbudakguy I think we can close!
